governance/process.md

```diff
@@ -1,144 +1,209 @@
+     outcome, ideas that they have for how it might happen, and ask if others
+     have ideas or questions.
    * After a short discussion, people should be invited to chime in on the
-   github issue and also mention of they
-   are interested in collaborating. This ensures that solutions are created
-   with multiple perspectives as well as verifies there is community interest
-   and energy to work on the proposal.
+     github issue and also mention of they are interested in collaborating. This
+     ensures that solutions are created with multiple perspectives as well as
+     verifies there is community interest and energy to work on the proposal.
 
    C) Discussion continues in the github issue for at least one week, usually
-   over multiple weeks or sometimes months.
+   over multiple weeks or sometimes months depending on the level of interest.
+   It is sometimes recommended to schedule an initial scoping meeting with
+   interested individuals to help refine the scope. We recommend using a doodle
+   poll or other polling mechanism in the slack channel to determine everyone's
+   availability. If interested in scheduling a meeting but don't have a
+   mechanism to conduct the meeting, reach out to the leadership team to have a
+   zoom meeting set up.
 
    The outcome of this conversation will be:
-    * Scope may be refined (or questions from the group may need follow-up
-    in order to define the scope)
+    * Scope may be refined (or questions from the group may need follow-up in
+      order to define the scope)
     * Criteria for completion are added to the issue that include a "definition
-    of done", ideally with validation by the target audience. Also note in the
-    issue if it will be a time-bounded project with a defined deliverable or
-    and on-going stream of work -- the latter would typically be proposed only
-    after at least one, usually multiple, projects have been completed.
-    * At least one person is recommended or nominated as a potential lead.
+      of done", ideally with validation by the target audience. Also note in the
+      issue if it will be a time-bounded project with a defined deliverable or
+      and on-going stream of work -- the latter would typically be proposed only
+      after at least one, usually multiple, projects have been completed.
+    * At least one person is recommended or nominated as a potential lead --
+      usually the author of the proposal.
     * Those interested in working on the solution comment on the issue so
-    coordination may begin and set up time or expectations with others to
-    begin work.
-    * A Chair or Technical Lead proposes this as a roadmap proposal or
-    agrees to act as a "sponsor."
-      * Sponsor: takes responsibility to ensure that progress
-      is tracked and that outcomes are reported to the group, including
-      proposing to close the issue if there is not sufficient activity to
-      sustain the effort.
```

Comment:
I dunno -- if no one is interested in sponsoring it, which typically happens if there's too much going on right now, do we want to encourage scoping/investment in getting the thing done?  In practice, I've rarely seen something well scoped without the collaboration of a Chair or TL

Reply:
IME - scoping is driven my the group or author with feedback from the Chair TL.  Doesn't mean it doesn't happen, and ultimately those conversations that drive scoping are usually with the chair/TL that is willing to sponsor (as sponsor they "sign off" on the scope as well?).  I've not seen where we don't sponsor due to timing, rather we sponsor but postpone the work.

Reply:
There are currently 21 [open proposals](https://github.com/cncf/tag-security/issues?q=is%3Aissue+is%3Aopen+label%3Aproposal) and I thought most were open due to lack of bandwidth. (If TL/Chairs weren't already working on so many things in parallel, then I figured some of those would be moving forward.)

Reply:
I think a lot of these no longer have interest.  its worth the time to sort through them and triage them accordingly.  thanks for bringing this up.